tools/upload-gh-pages.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Upload website to gh-pages
+USAGE="$0 <html_dir> <project-name> [<organization-name>]"
+HTML_DIR=$1
+if [ -z "$HTML_DIR" ]; then
+    echo $USAGE
+    exit 1
+fi
+PROJECT=$2
+if [ -z "$PROJECT" ]; then
+    echo $USAGE
+    exit 1
+fi
+ORGANIZATION=$3
+if [ -z "$ORGANIZATION" ]; then
+    ORGANIZATION=nipy
+fi
+upstream_repo="https://github.com/$ORGANIZATION/$PROJECT"
+cd $HTML_DIR
```

Comment:
Good point - sorry - I had already done this in the other projects, but forgot to upload these changes here.  How about now?